ecosystem/typescript/sdk/src/providers/aptos_client.ts

```diff
@@ -308,6 +316,91 @@ export class AptosClient {
     return builder.build(payload.function, payload.type_arguments, payload.arguments);
   }
 
+  /**
+   * Generates a fee payer transaction that can be signed and submitted to chain
+   *
+   * @param sender the sender's account address
+   * @param payload the transaction payload
+   * @param fee_payer the fee payer account
+   * @param secondary_signer_accounts an optional array of the secondary signers accounts
+   * @returns a fee payer raw transaction that can be signed and submitted to chain
+   */
+  async generateFeePayerTransaction(
+    sender: MaybeHexString,
+    payload: Gen.EntryFunctionPayload,
+    fee_payer: AptosAccount,
+    secondary_signer_accounts: Array<AptosAccount> = [],
```

Comment:
These accounts don't require private keys right?  Just the address, maybe the public key?

Reply:
yea... I think we should standardize the account input type through out the sdk. some functions ask for the account, some for the account address and some for a hex string. we should probably have one input type and a way to derive whatever the function needs within the sdk.

Reply:
In this case the e2e flow will often not include the signers so I think we should definitely use addresses, not accounts, I meant to start a discussion about this yesterday but forgot to hit send I guess

Reply:
Then can we take in an address then instead, or does it need more than the address?

Reply:
oh wait my bad, misreading, FeePayer expects an Array of `AccountAddress` and not `AptosAccount` 🤦‍♀️
 ill change the input to accept AptosAccount.

Reply:
I'm fairly sure the actual raw txn does not need anything other than the secondary addresses and the fee payer address. Obviously the order of the addresses matter, because this is the order of the `&signer`s in a multi-signer function call.

But yes, this transaction will need to be constructed in multiple contexts where the signers are signing independently of one another, so it only makes sense to use addresses here